### ISSUE TITLE
loss = nan

### ISSUE THREAD

**Issue by lyp0413** [Jittor/jittor#23]
I run the example of Linear Regression,but the loss became nan since step2.

**Comment by DuinoDu** [comment]
+1

**Comment by Gword** [comment]
Thanks for reporting, we updated the example. You need to modify the loss function from
```loss = ((pred_y - y)**2)``` 
to
```loss = jt.sqr(pred_y - y)```
The cause of this problem is that using gpu to calculate (-1) ** 2 will return nan. It should be noted that due to the different accuracy of the CPU and GPU, if you use the GPU to train the example, the loss may not reach the threshold.

**Comment by DuinoDu** [comment]
"**2" will be supported for gpu in future?

**Comment by Gword** [comment]
This is a known bug of nvidia: https://forums.developer.nvidia.com/t/powf-x-y-gives-nan/28094
If you must use `a^b`, please make sure `a` is a positive number.
